examples/examples_test.go

```diff
@@ -115,6 +115,14 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"redis-master-service":   &api.Service{},
 			"redis-slave-service":    &api.Service{},
 		},
+		"../examples/guestbook/guestbook_ns": {
```

Comment:
You introduced the following directories:
```
examples/guestbook-go/guestbook-go_v3/
examples/guestbook/guestbook_ns/
```
They should be consistent. Also, the `guestbook_ns` example no longer uses namespace, and `guestbook` doesn't really need to be in the subdirectory names.

Both examples should be tested here.